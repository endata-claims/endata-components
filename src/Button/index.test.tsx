import React from 'react'
import { shallow } from 'enzyme'

import Button from '.'

describe('Test Button', () => {
  const renderWrapper = () => shallow(<Button label='Button' />)

  describe('Snapshots', () => {
    it('should match snapshots with label', () => {
      expect(renderWrapper()).toMatchSnapshot()
    })
  })
})