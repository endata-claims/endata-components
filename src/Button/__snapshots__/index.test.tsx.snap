// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Button Snapshots should match snapshots with label 1`] = `
<button>
  Button
</button>
`;
